ios/NativeModules/TextileIPFS.h

```diff
@@ -9,6 +9,17 @@
 #import "React/RCTBridgeModule.h" // Required when used as a Pod in a Swift project
 #endif
 
+#import <React/RCTBridge.h>
+
 @interface TextileIPFS : NSObject <RCTBridgeModule>
   // Define class properties here with @property
+- (NSString *)_getHashData:(NSString *)hashPath error:(NSError**)error;
+@end
+
+@interface RCTBridge (TextileIPFS)
+/**
+ * The shared ipfs instance
+ */
+@property (nonatomic, readonly) TextileIPFS *ipfs;
```

Comment:
Here we extend the RN bridge object and add a property to retrieve the single instance of the Ipfs natvie module. The bridge is a shared object that's accessible from all other native modules and native UI components, therefore this allows us to access the IPFS node API from any native module or native UI component we create.